AzureADSample/AzureADSample/Service/Constants.cs

```diff
@@ -1,10 +1,4 @@
-﻿using System;
-using System.Collections.Generic;
-using System.Linq;
-using System.Text;
-using System.Threading.Tasks;
-
-namespace AzureADSample.Service;
+﻿namespace AzureADSample.Service;
 
 public static class Constants
```

Comment:
Think we need to remove the client id string and put a placeholder that the developer replaces with their own ID from Azure. Link here also to docs where they can find it.